### DISCUSSION TITLE
Statue of Liberty - "Welcome to SunflowerLand"

### DISCUSSION THREAD

**Discussion by Q4FOIL** [sunflower-land/sunflower-land#1295] [Art]
![Schermata 2022-08-01 alle 15 56 04](https://user-images.githubusercontent.com/58088988/182164042-1699030f-2bdc-4edb-88ff-3bab3141fe92.png)

![image](https://user-images.githubusercontent.com/58088988/182162482-d0311b24-87ea-47dc-9ec3-550657bbd166.png)
23x59px

![image](https://user-images.githubusercontent.com/58088988/182162794-44fb089b-9f55-4d9e-a995-a6fa68099ac9.png)
46x118px

**Require :**

**1 USA Flag
100 Stone
150 SFL**

**Comment by future2back** [comment]
Please stop doing this no usability NFT, this is annoying.

**Comment by Q4FOIL** [comment]
Are just ideas, i don't want to convict people to make this a NFT  

Thank you for reply ❤️

**Reply by future2back** [reply]
It's too cute, they'll do it.
